docs/docs/intro.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Welcome
+sidebar_position: 1
+---
+
+# Welcome Juno
+
+Let's discover **Juno in less than 5 min**.
+
+## What you'll need
+
+- [Golang](https://go.dev/doc/install) version 1.18 for build and run the project.
+- [Cairo-lang](https://www.cairo-lang.org/docs/quickstart.html) if you want to do `starknet_call` command.
+
+### Installing dependencies
+
+You can get all the dependencies running the next command:
+
+```bash
+$ make install-deps
+```
```

Comment:
Should this be moved to the contributing/engineering guidelines? I don't think `install-deps` installs golang or cairo-lang, which may be confusing for readers.

Reply:
This is just a beta for the documentation, we will change it when we add more functionalities to main